src/sqlfluff/dialects/dialect_oracle.py

```diff
@@ -28,6 +30,16 @@
             "prompt_command",
             r"PROMPT([^(\r\n)])*((?=\n)|(?=\r\n))?",
             CommentSegment,
+        ),
+        RegexLexer(
+            "double_execution_sign",
+            r"@@([^(\r\n)])*((?=\n)|(?=\r\n))?",
+           CommentSegment, 
+        ),
+        RegexLexer(
+            "execution_sign",
+            r"@@([^(\r\n)])*((?=\n)|(?=\r\n))?",
+           CommentSegment, 
```

Comment:
What's the difference between these two? Should one be just a single `@`?

Reply:
Yes, thank you

Reply:
That was going to be my next question! 😁

Reply:
Could we merge them both into the one test SQL file? Don’t see much value in them being separate when they’re basically there same?

Reply:
Done!